scripts/06_create_coda_files.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# Check that the correct number of arguments were provided.
+if [ $# -ne 9 ]; then
+    echo "Usage: sh 06_create_coda_files.sh <user> <data-root> <json-input-path> <flow-name> <variable-name> <coding-scheme> <is-multi-coded> <has-yes-no> <auto-cleaner>"
```

Comment:
As written, this file needs to be run multiple times with different configurations for each variable. I suggest changing this such that this file runs docker-run.sh repeatedly, with the flow-name, variable-name, coding-scheme etc. pre-configured for each run.

In other words, remove the variable-specific args and code them into repeated calls to docker-run.sh here.

Reply:
Changing all the scripts where this is applicable